css-layout-classes/style.css

```diff
@@ -0,0 +1,42 @@
+#container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+header {
+  font-size: 32px;
+  font-weight: bold;
+}
+
+img {
+  width: 100%;
+}
+
+#container .row:nth-child(2) .column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+#container .row:nth-child(3) .column-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+#container .row:nth-child(4) .column-one-half:first-child {
+  flex-basis: 66%;
+}
+
+#container .row:nth-child(4) .column-one-half:last-child {
+  flex-basis: 33%;
+}
+
+#container .row:nth-child(5) .column-one-half:first-child {
+  flex-basis: 33%;
+}
+
+#container .row:nth-child(5) .column-one-half:last-child {
+  flex-basis: 66%;
+}
```

Comment:
You can do this, but it'd be a lot simpler if you just targeted each column directly. For example: 

```
.column-two-thirds {
  flex-basis: calc(200% / 3);
}

.column-fourth {
  flex-basis: 25%;
}
````